model.py

```diff
@@ -55,9 +74,9 @@ class Setting(db.Model):
 
     __tablename__ = "settings"
 
-    setting_id = db.Column(db.Integer, autoincrement=True, primary_key=True)
-    setting_name = db.Column(db.String(100), unique=True)
-    setting_default_value = db.Column(db.String(100))
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(100), unique=True)
+    default_value = db.Column(db.String(100))
```

Comment:
It wasn't clear what this was for so I left it. But if you're using it to set the default value of some model fields you can do that on the column itself. For example:

`age = db.Column(db.Integer, default=18)`